src/IRrecv.h

```diff
@@ -86,6 +86,12 @@ typedef struct {
   uint8_t timeout;   // Nr. of milliSeconds before we give up.
 } irparams_t;
 
+#if __cplusplus >= 202002L
+typedef volatile irparams_t atomic_irparams_t;
+#else
+typedef volatile irparams_t atomic_irparams_t;
```

Comment:
Identical definitions?

Reply:
> Identical definitions?

For now. It doesn't compile when wrapped like `std::atomic<irparams_t>`, as then there is an issue with the members. Once I find out how to resolve that (or not) I either fix the definition, or I'll remove the `#if`.